vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -170,21 +176,34 @@ def call(Map pipelineParams) {
                         println("testRegionMatrix: $testRegionMatrix")
                         def jobs_names = testRegionMatrix*.job_name.toSet()
                         println("Jobs names: $jobs_names")
+                        def image_name = null
                         for (job_name in jobs_names) {
                             println("Job name: $job_name")
                             for (def entry in testRegionMatrix) {
-                                def version = null
-                                def sub_tests = []
-                                def region = null
+                                 def cloud_provider = entry.cloud_provider ?: 'aws'
+                                 def version = null
+                                 def sub_tests = []
+                                 def region = null
+                                 if (scylla_version == "master" && !image_name){
+                                    region = entry.region ?: 'us-east-1'
+                                    def output = sh(script: "./docker/env/hydra.sh list-images -c ${cloud_provider} -r ${region} -o text", returnStdout: true).trim()
```

Comment:
Please add the backed to argument calls (i.e. don't assume it would stay the default)

Reply:
@fruch this is cloud_provider, no?

Reply:
So it is passed. Looks like you did not see that, missed